syntax = "proto3";

service SenderCSV {
    // recieves token, team name, timestamp
    rpc SendFile(SendRequest) returns (SendResponse) {}
}

message SendRequest {
    string token = 1;
    string name = 2;
    string data = 3;
}

message SendResponse {
    string message = 1;
}